hubot/package.json

```diff
@@ -0,0 +1,28 @@
+{
+  "name": "lab-bot",
+  "version": "0.1.0",
+  "private": true,
+  "author": "Kevin McAnoy <kmcanoy@redhat.com>",
+  "description": "Originally designed to connect slack to jenkins to approve builds",
+  "scripts": {
+    "start": "bin/hubot --adapter slack"
+  },
+  "dependencies": {
+    "hubot": "^2.19.0",
+    "hubot-diagnostics": "0.0.2",
+    "hubot-google-images": "^0.2.7",
+    "hubot-google-translate": "^0.2.1",
+    "hubot-help": "^0.2.2",
+    "hubot-heroku-keepalive": "^1.0.3",
+    "hubot-maps": "0.0.3",
+    "hubot-pugme": "^0.1.1",
+    "hubot-redis-brain": "0.0.4",
+    "hubot-rules": "^0.1.2",
+    "hubot-scripts": "^2.17.2",
+    "hubot-shipit": "^0.2.1",
+    "hubot-slack": "^4.4.0"
+  },
+  "engines": {
+    "node": "0.10.x"
```

Comment:
I'm inclined to keep since I didn't create this package.json on my own. From the doc you linked to it appears that syntax is only advisory.

`Unless the user has set the engine-strict config flag, this field is advisory only and will only produce warnings when your package is installed as a dependency.
`

Reply:
ok